ci-operator/config/openshift/cluster-kube-storage-version-migrator-operator/openshift-cluster-kube-storage-version-migrator-operator-release-4.5.yaml

```diff
@@ -17,9 +17,6 @@ images:
 - dockerfile_path: images/ci/Dockerfile
   from: base
   inputs:
-    bin:
-      as:
-      - registry.svc.ci.openshift.org/ocp/builder:golang-1.13
```

Comment:
@sanchezl @sttts your line is wrong, pls fix it in the Dockerfile https://github.com/openshift/cluster-kube-storage-version-migrator-operator/blob/master/images/ci/Dockerfile#L1